app/tests/reader_studies_tests/resources/ground_truth_new.csv

```diff
@@ -1,4 +1,4 @@
-images,foo,bool,choice,mchoice
-im1,'"no"','false','"option2"','["option0"]'
-im2,'"no"','false','"option1"','["option1", "option2"]'
-im3;im4,'"yes"','true','"option1"','["option1", "option2"]'
+images,foo,bool,choice,mchoice,foo_explanation
+im1,'"no"','false','"option2"','["option0"]',new explanation
```

Comment:
What happens when there is a comma in the explanation?

Reply:
Good point, I'll add the same quotation scheme used in other fields here as well.